apps/chat2/chat2.nim

```diff
@@ -370,10 +370,10 @@ proc readInput(wfd: AsyncFD) {.thread, raises: [Defect, CatchableError].} =
     discard waitFor transp.write(line & "\r\n")
 
 {.pop.} # @TODO confutils.nim(775, 17) Error: can raise an unlisted exception: ref IOError
-proc processInput(rfd: AsyncFD) {.async.} =
+proc processInput(rfd: AsyncFD, rng: ref HmacDrbgContext) {.async.} =
```

Comment:
ideally this type would be aliased to something like `RngCtx`, but currently it does not justify a new file and import everywhere.

Reply:
I thought the same 👍🏼